spec/cucumber/formatter/legacy_api/adapter_spec.rb

```diff
@@ -1703,7 +1703,7 @@ def apply_before_hooks(test_case)
             end
 
             expect( formatter.legacy_messages ).to eq([
-                :before_features,
+                                                        :before_features,
                   :before_feature,
```

Comment:
When I was making my recent PRs with Rubocop fixes I opted not to change the indentation in this file and to keep it excluded for the reason @nodo gave.